bot/gerrit.go

```diff
@@ -26,41 +26,76 @@ type (
 			} `json:"commit"`
 		} `json:"revisions"`
 	}
+
+	goCL struct {
+		Tweeted   bool `datastore:",noindex"`
+		CrawledAt time.Time
+	}
```

Comment:
nit: I understand it's not required, but may want to consider giving an explicit `datastore:"name"` to be explicit and prevent issues if the struct field names are ever refactored.